File: Python/SimpleMethod.py
import math
class SimpleMethod:
    
    def CetakJudul(self):
        Title01 = 'Cetak judul dari class method'
        Title02 = 'Method ini dipanggil dari main program'
        print(Title01)
        print(Title02)

    def HitungLuasPersegi(self):
        print('\nMenghitung Luas Persegi dari method')
        Panjang = 10
        Lebar = 4
        Luas = Panjang * Lebar
        print('Luas Persegi Panjang = %2.2f'%Luas)

    def HitungLuasLingkaran(self):
        print('\nMenghitung Luas Lingkaran dari method')
        r = input('masukan jari-jari = ')
        LuasLingkaran = math.pi * float(r) * float(r)
        print('Luas Lingkaran = %5.2f'%LuasLingkaran)

Judul01 = 'Cetak judul tanpa class method'
Judul02 = 'judul02 dicetak di sini'
print(Judul01)
print(Judul02)
print('\n')
myMethod = SimpleMethod()
myMethod.CetakJudul()
myMethod.HitungLuasPersegi()
myMethod.HitungLuasLingkaran()

File: Simple Input Output/src/SimpleMethod.java
import java.util.Scanner;

public class SimpleMethod {

    public static void CetakJudul(){
        String Title01;
        String Title02;
        Title01 = "Mencetak Judul dalam method";
        Title02 = "Method cetakjudul terpanggil dari public void main";
        System.out.println("\n"+Title01);
        System.out.println(Title02);
    }

    public static void HitungLuasPersegi() {
        double Panjang;
        double Lebar;
        double Luas;
        Panjang = 10;
        Lebar = 15;
        Luas = Panjang * Lebar;
        System.out.println("\n Luas Persegi Panjang = "+Luas);
    }

    private static void HitungLuasLingkaran(){
        Scanner inputUser = new Scanner(System.in);
        double jarijari;
        double LuasLingkaran;
        System.out.println("\n Masukan Jarijari = ");
        jarijari = inputUser.nextDouble();
        LuasLingkaran = Math.PI* jarijari*jarijari;
        System.out.println("Luas Lingkaran = " + LuasLingkaran);
    }

    public static void main(String[] args) {
        String Judul01;
        String Judul02;

        Judul01 = "cetak kalimat tanpa method";
        Judul02 = "Ini adalah contoh untuk mencetak judul tanpa method";
        System.out.println(Judul01);
        System.out.println(Judul02);

        // call method 'cetakjudul' at line 3
        CetakJudul();

        // Menghitung luas persegi panjang dengan memanggil method hitungluaspersegi
        HitungLuasPersegi();

        // menghitung luas lingkaran dengan memanggil method hitung luasLIngkaran
        HitungLuasLingkaran();

    }


}
